cylc/flow/rundb.py

```diff
@@ -251,6 +251,7 @@ class CylcWorkflowDAO:
             ["flow_nums"],
             ["is_manual_submit", {"datatype": "INTEGER"}],
             ["try_num", {"datatype": "INTEGER"}],
+            # This is used to store simulation task start time across restarts.
             ["time_submit"],
```

Comment:
?

Reply:
Oliver and I identified that this is no longer used for whatever it may have been used for in the past - so we don't have to make db changes. However, since it's not obvious that this is now used for an entirely different function a comment seems warranted.